data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+				`	"Why would they want to join you?"`
+					goto why
+				`	"No way. I'm not helping your crusade."`
+					decline
+			`	"I like your enthusiasm, but all things most come in good time. First, you need to bring these people to their new home."`
+			label details
+			`	You were readying to leave but he stops you, "Oh and before I forget, we have divided the contract in two for practical reasons, you may choose to first get the Hai South, or North. You could even do both at the same time, provided you have enough bunks for that. Be mindfull that some could be impatient, don't keep them waiting too long else they may take the services of another captain, or worse, change their minds about joining us! They should at least wait a month or two, but if for some reason, some of them do not return to us, you will only be payed part of the price."`
+				accept
+
+	to complete
+		has "Unfettered: The Word Has Spread North: done"
+		has "Unfettered: The Word Has Spread South: done"
+	on complete
+		log `Gathered all the Hai who wanted to join the Unfettered that I could find.`
+		clear "unfettered: days"
+
+mission "Unfettered: Refresh The Clock"
+	invisible
+	repeat
+	to offer
+		has "Unfettered: The Word Has Spread: active"
+	on offer
+		"unfettered: days" = day + month * 30 + year * 365
+
+mission "Unfettered: The Word Has Spread North"
+	name `Gather the true Hai North`
+	description `Now that the Hai know of the Unfettered plan, a lot of them are eager to join them, while most cower in fear. Bring Hai back from <stopovers>, to <destination>.`
+	source "Firelode"
+	passengers 72
+	blocked `You need <bunks> passenger space for the next Unfettered mission.`
+
+	stopover "Cloudfire"
+	stopover "Snowfeather"
+	stopover "Mirrorlake"
+	stopover "Mahoganybox"
+	stopover "Makerplace"
+	stopover "Darkmetal"
+	stopover "Stonebreak"
+
+	on stopover
+		dialog `You have gathered all the Hai that had been assigned to you for the North part of Hai space, time to bring them to <destination>.`
+
+	to offer
+		has "Unfettered: Solifuge 5: Materials For Ships: done"
+
+	on offer
+		"unfettered: startdays north" = day + month * 30 + year * 365
+	
+	on visit
+		dialog phrase "generic missing stopover or passengers"
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on complete
+		conversation
+			branch "reduced payment"
+				"unfettered: missing northern hai" > 1
+			action
+				payment 3000000
+			`The Hai from up North are all excited to join the Unfettered. You receive your payment of <payment>.`
+				decline
+			label "reduced payment"
+			branch "very reduced payment"
+				"unfettered: missing northern hai" > 2
+			action
+				payment 2000000
+			``
+				goto pay
+			label "very reduced payment"
+			branch "no payment"
+				"unfettered: missing northern hai" > 3
+			action
+				payment 1000000
+			label pay
+			`The Hai from up North are all excited to join the Unfettered. You only receive <payment> as a payment because not all Hai as accounted for.`
```

Comment:
```suggestion
			`The Hai from up North are all excited to join the Unfettered. You only receive <payment> as a payment because not all Hai are accounted for.`
```